Makefile

```diff
@@ -45,7 +45,8 @@ fmt:
 	go fmt ./...
 
 generate:
-	go install ./mdtogo
+	go install ./mdtogo 
+	(which swagger || go install github.com/go-swagger/go-swagger/cmd/swagger@latest)
```

Comment:
avoid using `latest` tag, pin it to the latest stable version.

Reply:
Done, pinned to 0.27.0